doc/releases/mimic.rst

```diff
@@ -5,6 +5,14 @@ This is the second bugfix release of the Mimic v13.2.x long term stable release
 series. This release contains many fixes across all components of Ceph.
 We recommend that all users upgrade.
 
+* http://tracker.ceph.com/issues/36686 is an upgrade bug in this version,
+  where a partial upgrade during recovery/backfill, can cause the OSDs to
+  fail. The workaround for users is to upgrade and restart all OSDs or only
+  upgrade when all PGs are active+clean.
```

Comment:
@neha-ojha Does this slightly rephrased version sound good to you? 

* This version contains an upgrade bug, http://tracker.ceph.com/issues/36686, due to which upgrading during recovery/backfill can cause OSDs to fail. This bug can be worked around, either by restarting all the OSDs after the upgrade, or by upgrading when all PGs are in "active+clean" state.

Reply:
@smithfarm updated!